src/styles.scss

```diff
@@ -562,4 +562,10 @@ input.verify-sucess {
 a.highlight {
   color: $color;
   text-decoration: none;
-}
\ No newline at end of file
+}
+
+.qr-code {
+  img {
+    display: inline !important;
```

Comment:
This all belongs into `src/assets/theme.scss`

Reply:
Okay let me quickly move it